app/react/Library/Library.js

```diff
@@ -21,13 +20,7 @@ export default class Library extends RouteHandler {
     this.zoomOut = () => wrapDispatch(dispatch, 'library')(zoomOut());
   }
 
-  static renderTools() {
-    return (
-      <>
-        <ImportProgress />
-      </>
-    );
-  }
+  static renderTools() {}
```

Comment:
Without `Import progress` why are you keeping this definition?

Reply:
In `App` component, the function is called in a loop of many components, and all components looped are expected to have the function, otherwise there is a `function is undefined` error